### DISCUSSION TITLE
freac 1.1.7 in Mac OS Ventura (M2 silicon) crashes when I put in a CD

### DISCUSSION THREAD

**Discussion by hexatrope** [enzo1982/freac#537] [Help & Support] [answered]
I am using a LG WH14NS40 Blu ray drive in an external housing with my M2 Mac Mini, running Mac OS Ventura. Each time I put an audio CD in while freac is running, the application crashes. Anyone know how to fix this?

**Comment by Lee-Carre** [comment, marked as answer]
See #530 (and, originally, #457).

In a nutshell; use the [Continuous Build](/enzo1982/freac/releases/tag/continuous), for now (until v1·1·8+ is released).

**Comment by hexatrope** [comment]
Thank you. This fixed it.

**Reply by kokosowy** [reply]
Can you recheck? I've just used latest continuous build and it seems the issue is still there.

**Reply by mettkalf** [reply]
I did on a Mac mini M1, MacBook Air M2 and MacBook Air M3. As soon as I add an audio CD to the job list freac just closes and gives me an error message. 1.1.6 was the last version that worked for me.

**Comment by jdam77** [comment]
The same here on M2 pro with Sequoia 15.1.1, crash on reading CD both with version 1.1.7 and continuous build.

**Reply by mettkalf** [reply]
I tried it with the latest continous build (December 4th) again yesterday. Still crashes while adding an audio CD to the joblist.
Even 1.1.6 is unusable in my case since MacOS 15 (or 15.1) when "ripping" an audio CD. I get pops every 0.5 seconds.
So I rip CDs with Apple Music and encode the files into a single mp3 file with freac. But since 15 or 15.1 listing to the the mp3 via previev (space bar) it says that there is an error reading the file.
I have no idea what's going on...

**Comment by The-QA-Geek** [comment]
I just tested the continuous build from 'three weeks ago', so about October 1st, running on an M1 MacBook Pro and macOS 15.7.1 (Sequoia). It worked fine - no crashes and the rip sounds fine (no pops). The standard 1.1.7 at https://github.com/enzo1982/freac/releases/download/v1.1.7/freac-1.1.7-macos11.dmg crashes every time I put an audio CD into the drive.

Thank you to whoever fixed this, and to @enzo1982 for the new build. And, since I have used freac many times, I took this opportunity to send my appreciation via PayPal.
